src/main/java/com/bobocode/svydovets/annotation/annotations/Qualifier.java

```diff
@@ -1,7 +1,23 @@
 package com.bobocode.svydovets.annotation.annotations;
 
-//todo():provide JavaDocs
-//todo():add target and retention
-//todo():integrate with post processor
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * Annotation that can be used on a field, method, or constructor parameter to indicate a specific bean should be
+ * autowired.
+ *
+ * <p>The bean is selected by its qualifier attribute, which is specified through the value parameter of this
+ * annotation.
+ *
+ * <p>If a qualifier value is not specified, the default value is used.
+ *
+ */
+@Target({ElementType.CONSTRUCTOR, ElementType.METHOD, ElementType.FIELD})
```

Comment:
I think that instead of `ElementType.CONSTRUCTOR` we should keep `ElementType.PARAMETER`.
We will mark specific constructor parameters by this annotation, not the whole one.